build.gradle

```diff
@@ -18,8 +18,8 @@ allprojects {
 subprojects {
     apply plugin: 'java'
     apply plugin: 'jacoco'
-    sourceCompatibility = "9"
-    targetCompatibility = "9"
+    sourceCompatibility = "8"
+    targetCompatibility = "8"
```

Comment:
Is there any difference in using "1.8" vs "8"?

Reply:
Gradle (actually I think Java) seems good with both - we've been using "8" in the Spark connector for months now.

Reply:
ahh okay